src/app/core/password-recovery/password-recovery.service.ts

```diff
@@ -48,4 +53,21 @@ export class PasswordRecoveryService {
         )
       )
   }
+
+  resetPasswordEmail(resetPassword: ResetPasswordEmailForm) {
+    let encodedData = JSON.stringify(resetPassword)
+    return this._http
+      .post<ResetPasswordEmailForm>(
+        environment.API_WEB + `reset-password-email-v2.json`,
```

Comment:
@leomendoza123 is this the correct endpoint to use?

Reply:
yes